docs/maintain-guides-how-to-validate-polkadot.md

```diff
@@ -0,0 +1,392 @@
+you should review the output from the "git tag" command (`git tag | grep "$v\0\.7"`) to see a list
+of all the potential 0.7 releases. You should replace `v0.7.28` with the latest build (i.e., the
+highest number). You can also find the latest Kusama version on the
+[release](https://github.com/paritytech/polkadot/releases) tab.
+
+> Note: If you prefer to use SSH rather than HTTPS, you can replace the first line of the below with
+> `git clone git@github.com:paritytech/polkadot.git`.
+
+```sh
+git clone https://github.com/paritytech/polkadot.git
+cd polkadot
+git tag | grep "$v\0\.7"
+git checkout v0.7.28
+./scripts/init.sh
+cargo build --release
+```
+
+This step will take a while (generally 15 - 30 minutes, depending on your hardware).
+
+If you are interested in generating keys locally, you can also install `subkey` from the same
+directory. You may then take the generated `subkey` executable and transfer it to an air-gapped
+machine for extra security.
+
+```sh
+cargo install --force --git https://github.com/paritytech/substrate subkey
+```
+
+### Synchronize Chain Data
+
+> **Note:** Validators must sync their nodes in archive mode to avoid being slashed. If you've
+> already synced the chain, you must first remove the database with `polkadot purge-chain` and then
+> ensure that you run Polkadot with the `--pruning=archive` option.
+
+You can begin syncing your node by running the following command:
+
+```sh
+./target/release/polkadot --pruning=archive
+```
+
+if you do not want to start in validator mode right away.
+
+**Note:** The `--pruning=archive` flag is implied by the `--validator` and `--sentry` flags, so it
+is only required explicitly if you start your node without one of these two options. If you do not
+set your pruning to archive node, even when not running in validator and sentry mode, you will need
+to re-sync your database when you switch.
+
+Depending on the size of the chain when you do this, this step may take anywhere from a few minutes
+to a few hours.
+
+If you are interested in determining how much longer you have to go, your server logs (printed to
+STDOUT from the `polkadot` process) will tell you the latest block your node has processed and
+verified. You can then compare that to the current highest block via
+[Telemetry](https://telemetry.polkadot.io/#list/Kusama) or the
+[PolkadotJS Block Explorer](https://polkadot.js.org/apps/#/explorer).
+
+> **Note:** If you do not already have DOTs, this is as far as you will be able to go until the end
+> of the soft launch period. You can still run a node, but you will need to have a minimal amount of
+> DOTs to continue, as balance transfers are disabled during the soft launch. Please keep in mind
+> that even for those with DOTs, they will only be indicating their _intent_ to validate; they will
+> also not be able to run a validator until the NPoS phase starts.
+
+## Bond DOTs
+
```

Comment:
Add a note here that until transfers are enabled, you _must_ use the same account if you did not get DOT injections done.